persist-cli/src/main/java/io/ballerina/persist/cmd/PersistCodeGeneratorTool.java

```diff
@@ -76,6 +81,14 @@ public void execute(ToolContext toolContext) {
             }
             validatePersistDirectory(datastore, projectPath);
             printExperimentalFeatureInfo(datastore);
+            try {
+                if (validateCache(toolContext, schemaFilePath)) {
+                    errStream.println("INFO: no changes in the schema file. Generated sources are up to date.");
```

Comment:
will this be annoying as each time we build, this will print? is this the same in openAPI tool?

Reply:
I prefer printing only if regenerate the client code.

Reply:
Fixed in e24c4bc